contributors/guide/README.md

```diff
@@ -95,20 +95,13 @@ _Improvements needed_
 ### Find a SIG that is related to your contribution
 
 You may have noticed that some repositories in the Kubernetes Organization are owned by Special Interest Groups, or SIGs. We organize the Kubernetes community into SIGs in order to improve our workflow and more easily manage what is a very large community project.
-SIGs also have their own CONTRIBUTING.md files, which may contain extra information or guidelines in addition to these general ones.
+SIGs also have their own CONTRIBUTING.md files, which may contain extra information or guidelines in addition to these general ones.  These are located in the SIG specific community documentation directories, for example: sig-docs' is in [/community/sig-docs/CONTRIBUTING.md](/community/sig-docs/CONTRIBUTING.md) and similarly for other SIGs.
 
-After finding something to contribute, you should find the appropriate SIG, which you will need in order to have your pull request approved for testing and merging.
-
-Here is how (_information needed_)
+After finding something to contribute, you should find the appropriate SIG, which you will need in order to have your pull request approved for testing and merging.  See [Open A Pull Request](#open-a-pull-request) below.
```

Comment:
This is a great way to find out about SIGs in an organic way. Is there any way we can make it easier to determine the SIG _before_ opening a PR? Although honestly I think this is pretty good for now.